sites/main-site/src/content/hackathon-projects/hackathon-march-2025/aus-hpc-training.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Nextflow for HPC training in Australia
+category: components
+slack: ""
```

Comment:
Suggestion - use the `#training` channel in the nf-core slack as the main channel?

```suggestion
slack: https://nfcore.slack.com/archives/CP1T5S815
```